### DISCUSSION TITLE
Sentence Case vs Title Case

### DISCUSSION THREAD

**Discussion by a-christensen** [twilio-labs/paste#417] [General] [answered]
Hello team! 👋

When trying to implement the new sentence case guidelines in Paste, I ran into some areas that need clarification / additional considerations.

1. **The Navigation + Page Titles**: If products are capitalized, this means that some areas of the navigation along with some page titles will use sentence case and others won't. This could look inconsistent even though it is following logic. Are we okay with this? (see [mockup](https://share.goabstract.com/ae72a28e-5420-424a-a6ad-774a48e9a2be?mode=design&sha=f6b659e4fd2e42adea3d9b400c984cfb36ce6886), specifically "Messaging Insights" vs "Message queues."

2. **Twilio Term Ambiguity**: There is ambiguity around what is considered "features, programs, or systems that Twilio has given a specific name to." Would this apply to the "Passcode Conversions" tab (see [mockup](https://share.goabstract.com/ae72a28e-5420-424a-a6ad-774a48e9a2be?mode=design&sha=f6b659e4fd2e42adea3d9b400c984cfb36ce6886) for the tabs under the filter)? That dashboard is not branded but it is a specific name that Twilio is giving these kinds of messages (a Twilio term). How would this apply to the term "Twilio numbers"? Twilio numbers are products so should they be in title case like "Twilio Numbers"? Or because "numbers" is a generic term should this be sentence case like "Twilio numbers"?

3. **The Tab Titles**: If "Passcode Conversions" should be title case, does this look inconsistent with the "Delivery & errors" tab? Will this tab name look like an oversite or unfinished to customers, since it's the only one in sentence case?

**Comment by SiTaggart** [comment]
Going to ping @ctecks on this one as they would be the best person to give the first answer to this.

**Comment by ctecks** [comment, marked as answer]
Hi @a-christensen thanks for the queries.

tl;dr sentence case everywhere will take time to get used to for us in the short term, but provides a more accessible, scalable experience in the long term. Ambiguities about capitalizing specific terms are welcome - please send them through and I'll include them in the guidance. 

1. From a content readability, consistency, and accessibility standpoint, yes we are ok with this. The guidance to use sentence case everywhere in product content stands. There is going to be a period of transition, implementing the new guidance while legacy pages still use title case - this will give rise to some inconsistency between new and old for a time, however the same applies to pages using existing styling vs new ones built with Paste. 

As regards things 'looking inconsistent', it's important that we're consistent with the guidance, even though it looks unfamiliar to us now - the change will take us as a UX Team a little time to get used to, however in the long-term we are laying the foundation for more consistent, understandable content and UI across all products. I'm keen to highlight (and will add to the guidance) that using sentence case throughout supports 

- brand voice and tone (human, conversational)
- localization (different languages have different capitalization rules - sentence case enables localization while retaining clarity and intent)
- readability (sentence case generally reduces cognitive load and enables users to scan for products and terms they need to complete their task)
- consistency at scale (applying sentence case throughout should enable every PD to work with less ambiguity - although I fully acknowledge that there are ambiguities requiring clarification at present) See 2.

2. These ambiguities (e.g. is this a generic term, or a Twilio term?) are exactly the kinds of things that the style guide aim to provide guidance with - please do keep sending them through, because this will help me to help you define what does and doesn't take a capital. I'm trying to compile a list of contentious queries, such as 'Twilio Phone Number' Vs 'phone number' to provide clarity and rely on you and your PD colleagues to share these - thank you for doing so. 

In the short term, my guidance is to use 'Twilio phone number' and 'phone number' or 'number' elsewhere. Only capitalize 'phone number' if it's part of a product/feature name "Phone Number API."

3. The example you shared looks great to me, because the tab name are consistent within the context of the page. If we were using title case for nav items and page headers, but sentence case for tab names, then it would have potential to look confused or unfinished. In that particular instance, I wonder if there's space for "Delivery and errors", replacing the ampersand, though I appreciate that space could be an issue. 

Very happy to continue the conversation, and thank you for raising the need for clarification in this section (and providing some examples to help me do so) - the next release should be along in a sprint or two's time, and I'll be sure that these items are included to meet needs. Please keep the feedback coming, it's invaluable.
